### PR TITLE
Fix a few more occurrences of divisive language

### PR DESCRIPTION
**Pull request by markusthoemmes**
As per title.

/assign @nak3 @tcnghia

### CODE REVIEW

File: pkg/status/status_test.go [hunk 1]
```diff
@@ -191,17 +191,17 @@ func TestProbeLifecycle(t *testing.T) {
 		}
 	}()
 
-	// Dummy handler returning HTTP 500 (it should never be called during probing)
-	dummyRequests := make(chan *http.Request)
-	dummyHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		dummyRequests <- r
+	// Test handler returning HTTP 500 (it should never be called during probing)
```
Comment (JRBANCEL) [inline pkg/status/status_test.go:194]: This might be well intentioned, but `test` is super vague.

Comment (vagababov) [reply pkg/status/status_test.go:194]: `failingHandler`

Comment (markusthoemmes) [reply pkg/status/status_test.go:194]: point taken, I'll revisit!

**Review by vagababov — COMMENTED** [review]
/lgtm

**Pull request merged**